File: wrapper/ios/Feature/Entity/Box.swift
public struct Box {
    public let xmin: Float
    public let ymin: Float
    public let xmax: Float
    public let ymax: Float
    
    public init(xmin: Float, ymin: Float, xmax: Float, ymax: Float) {
        self.xmin = xmin
        self.ymin = ymin
        self.xmax = xmax
        self.ymax = ymax
    }
}

File: wrapper/android/ext/src/main/java/com/zeticai/mlange/feature/entity/Box.kt
package com.zeticai.mlange.feature.entity

data class Box(
    var xMin: Float,
    var yMin: Float,
    var xMax: Float,
    var yMax: Float
) {
    fun reset(xMin: Float, yMin: Float, xMax: Float, yMax: Float) {
        this.xMin = xMin
        this.yMin = yMin
        this.xMax = xMax
        this.yMax = yMax
    }
}
